bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/DeviceManagerWrapper.java

```diff
@@ -39,6 +39,7 @@ public DeviceManagerWrapper(@Nullable DeviceManager deviceManager) {
         this.deviceManager = deviceManager;
     }
 
+    @SuppressWarnings("null")
```

Comment:
Is it necessary to hide warnings?

Reply:
Well, define necesary :-)  It is needed because it is a `synchronized` with a good null check on `deviceManager` but the compile throws a false postive warning about possible NPE on this variable

So it is only needede to satisfy the compiler, on runtime it won't matter.